sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerBuilder.java

```diff
@@ -215,6 +217,36 @@ public SchedulerBuilder setPlanCustomizer(PlanCustomizer planCustomizer) {
         return this;
     }
 
+    public SchedulerBuilder withSingleRegionConstraint() {
```

Comment:
API looks good. Subjective: Maybe defer the actual serviceSpec manipulation until `build()`? This could do something like set a boolean to true, which then gets checked for in `build()`.